src/manager.js

```diff
@@ -1,72 +1,132 @@
+import { requireLoader } from "@jupyter-widgets/html-manager";
+
 import * as pWidget from "@lumino/widgets";
 
-import { HTMLManager } from "@jupyter-widgets/html-manager";
+import {
+  RenderMimeRegistry,
+  standardRendererFactories,
+} from "@jupyterlab/rendermime";
 
-import * as outputWidgets from "./output";
+import {
+  WidgetManager as JupyterLabManager,
+  WidgetRenderer,
+  output,
+} from "@jupyter-widgets/jupyterlab-manager";
 
-import { ShimmedComm } from "./services-shim";
+import * as base from "@jupyter-widgets/base";
 
-export class ThebeManager extends HTMLManager {
-  get onError() {
-    return this._onError;
-  }
+import * as controls from "@jupyter-widgets/controls";
 
-  registerWithKernel(kernel) {
-    if (this._commRegistration) {
-      this._commRegistration.dispose();
-    }
-    this._commRegistration = kernel.registerCommTarget(
-      this.comm_target_name,
-      (comm, message) => this.handle_comm_open(new ShimmedComm(comm), message)
-    );
-    this.kernel = kernel;
+const WIDGET_MIMETYPE = "application/vnd.jupyter.widget-view+json";
+
+export class ThebeManager extends JupyterLabManager {
```

Comment:
What are the reasons the methods `build_widgets()`, `_build_models()`, `_update_comm()`, and `createContext()` aren't present? I see those in: https://github.com/voila-dashboards/voila/pull/46/files#diff-cee1519068dafb3b4c08abaad8890560d739e46fe1a85344c21457ebe63d5808